platforms/android/library/src/main/java/io/element/android/wysiwyg/EditorEditText.kt

```diff
@@ -276,6 +279,16 @@ class EditorEditText : AppCompatEditText {
         htmlConverter = createHtmlConverter(styleConfig, mentionDisplayHandler)
     }
 
+    fun setOnRichContentSelected(onRichContentSelected: ((Uri) -> Unit)?) {
+        if (onRichContentSelected != null) {
+            ViewCompat.setOnReceiveContentListener(
+                this,
+                arrayOf("image/*"),
```

Comment:
`arrayOf("image/*")` could be a parameter too, but I think it does not worth it.